### DISCUSSION TITLE
Concatenating two data columns in display

### DISCUSSION THREAD

**Discussion by Mike-Carolan** [Budibase/budibase#5213] [Help] [answered]
Hi, I was wondering if anyone has successfully concatenated two data columns into one for display purposes (i.e. f_name and l_name data columns into "Name" column for display purposes).  Thanks in advance!

**Comment by xnohat** [comment, marked as answer]
You can create a "formula" column and combine data from 2 column f_name and l_name

**Reply by melohagan** [reply]
Brilliant, good to know! Here's a screenshot for good measure:

<img width="379" alt="Screenshot 2022-04-02 at 21 39 46" src="https://user-images.githubusercontent.com/101575380/161400443-16e2db27-da20-4550-a023-09dafdb9e515.png">
